examples/__tests__/use-counter.js

```diff
@@ -0,0 +1,46 @@
+import {testHook, cleanup} from 'react-testing-library'
```

Comment:
This should be called `react-hooks` so the test is more about the APIs and less about the simulated component (similar to react-context etc.)

Reply:
done.